emission/net/api/cfc_webapp.py

```diff
@@ -379,6 +382,41 @@ def getCarbonCompare():
     logging.debug("No overriding client result for user %s, returning choice" % user_uuid)
   return choice.getResult(user_uuid)
 
+
+#Pulling public data from the server  
+@get('/getData')
+def getData():
+  from_date = request.query.from_date
+  to_date = request.query.to_date
+
+  from_dt = datetime.strptime(from_date, "%Y-%m-%d")
```

Comment:
nit: consider using the `arrow` library, which is much easier to use.
Then you can do:

```
from_ts = arrow.get(from_date).timestamp
```